src/helpers/safe.ts

```diff
@@ -29,7 +29,7 @@ async function convertTransactions(transactions: PreparedTransactionRequest[]) {
                 throw new Error('Invalid transaction')
 
             return {
-                to: await unwrapAddressLike(transaction.to),
+                to: getAddress(await unwrapAddressLike(transaction.to)),
```

Comment:
why use `getAddress`?

Reply:
addresses must be checksumed for safe to accept the transaction